main_app/templates/main_app/dog_form.html

```diff
@@ -1,4 +1,10 @@
-{% extends 'base.html' %} {% block content %}
+{% extends 'base.html' %} 
+{% block content %} 
+{% if object %}
+  <h1>Edit <span class="teal-text">{{object.name}}</span></h1>
+{% else %}
+  <h1>Add Dog</h1>
+{% endif %}
 
 <h1>Add a Dog</h1>
```

Comment:
This looks like it should be deleted (since it exists now in the else branch above).